.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 
 build_artefacts
+*.orig
```

Comment:
These files are generated from scratch each time and changes added to them will be removed. If you need to make changes to files outside the recipe, please raise an issue or send a PR to [conda-smithy]( http://github.com/conda-forge/conda-smithy ).